CONTRIBUTING.md

```diff
@@ -7,24 +7,31 @@ Getting started
 If you want to help but don't know where to start, here are some
 low-risk/isolated tasks:
 
-- [Merge a Vim patch].
 - Try a [good first issue](../../labels/good-first-issue) or [complexity:low] issue.
 - Fix bugs found by [Clang](#clang-scan-build), [PVS](#pvs-studio) or
   [Coverity](#coverity).
 - [Improve documentation][wiki-contribute-help]
+- [Merge a Vim patch]. [Runtime updates](https://github.com/vim/vim/search?q=update+runtime+files&type=commits) are easiest to get started.
 
 Reporting problems
 ------------------
 
+Before opening a new issue:
+
 - [Check the FAQ][wiki-faq].
 - [Search existing issues][github-issues] (including closed!)
 - Update Neovim to the latest version to see if your problem persists.
-- [Bisect](https://neovim.io/doc/user/starting.html#bisect) your config: disable plugins incrementally, to narrow down the cause of the issue.
-- [Bisect][git-bisect] Neovim's source code to find the cause of a regression, if you can. This is _extremely_ helpful.
+
+When reporting an issue:
+
+- [Bisect](https://neovim.io/doc/user/starting.html#bisect) your config: disable plugins and remove settings incrementally, to narrow down the cause of the issue.
+- Using this information, prepare a [minimal config](https://github.com/neovim/neovim/blob/master/contrib/minimal.lua) that reproduces the problem with `nvim -u minimal.lua`.
```

Comment:
this is too much. we can weakly suggest it, but to make it now the standard for bug reports means that everyone has to have this thing hanging around. most bug reports don't need this.

Reply:
The second motivation for this change is that leaving this open leads to many reports that just skip these steps even though they are needed for diagnosing the issue, forcing maintainers to request additional information.

Reply:
needing to install plugins should be rare for repro steps. huge burden for triagers. it is more useful if the repot narrows down the steps to actual code.

we should _rarely_ be debugging plugins, that massively increases the triage scope.

Reply:
Agreed, but this makes it _immediately_ obvious that (and which) plugins are involved instead of having to pull that information like a tooth.